docs/components/connectors/out-of-the-box-connectors/graphql.md

```diff
@@ -0,0 +1,212 @@
+
+### Configure Basic Authentication
+
+Select the **GraphQL Connector** and fill out the following properties under the **Authentication** section:
+
+1. Click **Basic** in the **Authentication** section.
+2. Set **Username** (i.e. `secrets.YOUR_USERNAME`).
+3. Set **Password** to the secret you created (i.e. `secrets.GRAPHQL_BASIC_SECRET`).
+
+![graphql Connector basic](../img/connectors-graphql-basic.png)
+
+### Bearer Token
+
+#### Create a new Connector secret
+
+We advise you to keep your **Bearer Token** safe and avoid exposing it in the BPMN `xml` file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `GRAPHQL_BEARER_TOKEN`) so you can reference it later in the Connector.
+
+#### Configure the Bearer Token
+
+Select the **GraphQL Connector** and fill out the following properties under the **Authentication** section:
+
+1. Click **Bearer Token** in the **Authentication** section.
+2. Set **Bearer** to the secret you created (i.e. `secrets.GRAPHQL_BEARER_TOKEN`).
+
+![graphql Connector bearer token](../img/connectors-graphql-bearer-token.png)
+
+### OAuth token
+
+#### Create a new Connector secret
+
+We advise you to keep your **OAUTH_TOKEN_ENDPOINT** safe and avoid exposing it in the BPMN `xml` file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `OAUTH_TOKEN_ENDPOINT`) so you can reference it later in the Connector.
+
+#### Configure the OAuth Token
+
+Select the **GraphQL Connector** and fill out the following properties under the **Authentication** section:
+
+1. Click **OAuth 2.0** in the **Authentication** section.
+2. Set **OAuth Token Endpoint** to the secret you created (i.e. `secrets.OAUTH_TOKEN_ENDPOINT`).
+3. Set **Client ID** to the secret you created (i.e. `secrets.CLIENT_ID`).
+4. Set **Client secret** to the secret you created (i.e. `secrets.CLIENT_SECRET`).
+5. (Optional) Set **Scopes** (i.e. `read:clients`). Depending on the OAuth provider you're using, this may or may not be required.
+6. Set **Audience** to the secret you created (i.e. `secrets.AUDIENCE`). It is an optional field. Depending on the OAuth provider you're using, you should fill this field or not.
+7. Choose **Client Authentication** from the dropdown menu (i.e. `Send client credentials in body`).
+
+![graphql Connector oauth token](../img/connectors-graphql-oauth-token.png)
+
+Find more information about the OAuth client credentials flow at the [RFC reference](https://www.rfc-editor.org/rfc/rfc6749#section-4.4).
+
+## HTTP Endpoint
+
+Under the **HTTP Endpoint** section, fill in the **URL** with your desired endpoint and select the desired **Method**.
+
+![graphql Connector method and url](../img/connectors-graphql-http-method-url.png)
+
+## GraphQL Query
+
+### Query/Mutation
```

Comment:
Does it make sense to also add an example here, how to use variables / FEEL to have a dynamic query?

Reply:
Currently we cannot have dynamic query. FEEL would not be able to handle the syntax of the GraphQL query.

Reply:
Ah ok. Thanks for explaining.